lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/net.rb

```diff
@@ -442,17 +442,21 @@ def cmd_portfwd(*args)
         print_line
 
       when 'add'
-
         if reverse
           # Validate parameters
           unless lport && lhost && rport
             print_error('You must supply a local port, local host, and remote port.')
             return
           end
 
+          unless rhost.nil?
+            print_warning('The remote host (-r) option is ignored for reverse port forwards.')
+          end
+
           begin
             channel = client.net.socket.create(
               Rex::Socket::Parameters.new(
+                'LocalHost' => '', # see: #17282, always bind to all interfaces
```

Comment:
Concerned if this will have to be ripped out once the other changes are implemented. Otherwise this code looks good.

Reply:
It would change if we wanted the LocalHost to be configurable for the `portfwd` command. After talking to @adfoster-r7, though, we don't want to change this command's behavior now and we don't have plans to do it in the future. It touched on this [here](https://github.com/rapid7/metasploit-framework/issues/17282#issuecomment-1341445522). Basically, the gist is we're concerned that users have come to rely on the fact that the argument is ignored for reverse port forwards and that changing it now to correct it's behavior may confuse those users. That's why a warning was added on L453, so the behavior is maintained but at least it's being called out now.